ml3d/datasets/semantickitti.py

```diff
@@ -139,7 +139,7 @@ def is_tested(self, attr):
     def save_test_result(self, results, attr):
         cfg = self.cfg
         name = attr['name']
-        name_seq, name_points = name.split("_")
+        name_seq, name_points = name.split("/")
```

Comment:
the '_' is used in the submission format for SemanticKITTI. Can we set this as an argument?